backends/platform/sdl/opendingux/miyoo.mk

```diff
@@ -0,0 +1,50 @@
+OD_EXE_STRIPPED := scummvm_stripped$(EXEEXT)
+bundle = sd-root
+
+all: $(OD_EXE_STRIPPED)
+
+$(OD_EXE_STRIPPED): $(EXECUTABLE)
+	$(STRIP) $< -o $@
+
+$(bundle): all
+	$(MKDIR) $(bundle)
+	$(MKDIR) $(bundle)/games
+	$(MKDIR) $(bundle)/games/scummvm
```

Comment:
`$(MKDIR) -p $(bundle)/games/scummvm` will create all the path in one command

Reply:
done